haiku-games/matchingcomulns/matching_columns-0.1.0.recipe

```diff
@@ -0,0 +1,42 @@
+SUMMARY="A matching game"
+DESCRIPTION="MatchColumns, a fun matching game."
+HOMEPAGE="https://github.com/spiceweasel/matching-columns"
+COPYRIGHT="2017 Scott Jacovidis"
+LICENSE="MIT"
+REVISION="1"
+srcGitRev="eac403db4a639b0893f47738896c60cb37f569c1"
+SOURCE_URI="https://github.com/spiceweasel/matching-columns/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="a042f9bc916a10332bd1475bd55af64cd71c28664afbfc333a4b57d558ad85ff"
+SOURCE_FILENAME="matching-columns-$srcGitRev.tar.gz"
```

Comment:
You should update this according to the [Guidelines][0]. The prefered format for this kind of package is `matchingcolumns-$portVersion-$srcGitRev.tar.gz`

[0]: https://github.com/haikuports/haikuports/wiki/HaikuPorter-Guidelines#archive-name